src/index.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response, Application } from 'express';
+import swaggerUi from 'swagger-ui-express';
+
+import { swaggerSpec } from './swagger/configs';
+
+import { healthCheckRouter } from './resources/health-check';
+import { aboutRouter } from './resources/about';
+import { abcdRouter } from './resources/ab-сd';
+
+export const app: Application = express();
+export const port = 3000;
```

Comment:
Lets move this variable to separate `config.ts` file because it also used in swagger module.

```suggestion
import { port } from './config';
```

`config.ts`:
```
export const port = 3000;
```

Reply:
Done